Resources/Prototypes/_NF/GameRules/roundstart.yml

```diff
@@ -4,6 +4,17 @@
   categories: [ HideSpawnMenu ]
   components:
   - type: AdventureRule
+    spaceDungeons:
+    - CaveFactory
+    - MedSci
+    - FactoryDorms
+    - LavaMercenary
+    - VirologyLab
+    - SalvageOutpost
+    # Upstream definitions
+    - Experiment
+    - SnowyLabs
+    - LavaBrig # Skipping Mineshaft, Haunted; they're in caves
```

Comment:
Can we make the frontier versions just NF ones?

Reply:
Just redefine it for now?  We're still using these for salvage dungeons as well.

Reply:
So be it, this is another PR issue.